src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -243,73 +291,97 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
           {showIsPublicPrivate && bundle.benefactorAcl && !isPublic(bundle) ? (
-            <LockTwoTone
-              aria-hidden={false}
-              role="img"
-              className="EntityBadge__Badge"
-              data-for={ownTooltipId}
-              data-tip={'Private'}
-              data-testid={'is-private-icon'}
-            />
+            <Tooltip title="Private" enterNextDelay={100} placement="right">
+              <LockTwoTone
+                aria-hidden={false}
+                role="img"
+                className="EntityBadge__Badge"
+                data-for={ownTooltipId}
+                data-tip={'Private'}
+                data-testid={'is-private-icon'}
+              />
+            </Tooltip>
           ) : null}
           {showHasLocalSharingSettings &&
           bundle.benefactorAcl &&
           entityId === bundle.benefactorAcl!.id ? (
-            <CheckTwoTone
-              aria-hidden={false}
-              role="img"
-              className="EntityBadge__Badge"
-              data-for={ownTooltipId}
-              data-tip="Sharing Settings have been set"
-              data-testid={'sharing-settings-icon'}
-            />
+            <Tooltip
+              title="Sharing Settings have been set"
+              enterNextDelay={100}
+              placement="right"
+            >
+              <CheckTwoTone
+                aria-hidden={false}
+                role="img"
+                className="EntityBadge__Badge"
+                data-for={ownTooltipId}
+                data-tip="Sharing Settings have been set"
+                data-testid={'sharing-settings-icon'}
+              />
+            </Tooltip>
           ) : null}
 
           {showHasAnnotations &&
             !!(annotationsCount || schemaValidationResults) && (
-              <LocalOfferTwoTone
-                aria-hidden={false}
-                role={canOpenModal ? 'button' : 'img'}
-                className={`EntityBadge__Badge ${schemaConformance}`}
-                style={canOpenModal ? { cursor: 'pointer' } : undefined}
-                onClick={canOpenModal ? () => setShowModal(true) : undefined}
-                data-for={ownTooltipId}
-                data-tip={sanitizeHtml(annotationsHtml)}
-                data-html={true}
-                data-testid={'annotations-icon'}
-              />
+              <Tooltip
+                title={annotationsHtmlX}
+                enterNextDelay={100}
+                placement="right"
+              >
+                <LocalOfferTwoTone
+                  aria-hidden={false}
+                  role={canOpenModal ? 'button' : 'img'}
+                  className={`EntityBadge__Badge ${schemaConformance}`}
+                  style={canOpenModal ? { cursor: 'pointer' } : undefined}
+                  onClick={canOpenModal ? () => setShowModal(true) : undefined}
+                  data-for={ownTooltipId}
+                  data-tip={sanitizeHtml(annotationsHtml)}
```

Comment:
Is this still used?  (data-tip, data-for, ...)

Reply:
Yes, it is.

Reply:
It looks like the actual tooltip comes from Tooltip.title.  Do all of these old values need to be set?

Reply:
Is annotationsHtml referenced after using annotationsHtmlX?

Reply:
> Is annotationsHtml referenced after using annotationsHtmlX?

No those aren't used. I must have forgotten to remove it.

Reply:
As for data-tip, data-for those are used for tests. Ones that don't I have removed.